contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -27,6 +27,7 @@
 class MypyTask(ResolveRequirementsTaskBase):
   """Invoke the mypy static type analyzer for Python."""
 
+  _MYPY_COMPATIBLE_INTERPETER_CONSTRAINT = '>=3.5'
```

Comment:
This might change over time with the version of mypy that is in use (which is itself configurable). It should probably move into an option next to the version.

Reply:
I would wait to make this an option until MyPy changes the minimum required version to not support 3.5. Keep it as simple as possible for now.

Reply:
Alrighty. Not a blocker.

Reply:
OK - thanks. I'll merge as-is and we'll fix when we hit an error due to mypy bump of the floor here.